catalog/src/main/kotlin/com/adevinta/spark/catalog/examples/samples/tokens/colors/ColorSample.kt

```diff
@@ -41,71 +43,88 @@ import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import com.adevinta.spark.SparkTheme
 import com.adevinta.spark.components.surface.Surface
+import com.adevinta.spark.components.text.Text
+import kotlin.reflect.KProperty0
 
 @OptIn(ExperimentalLayoutApi::class)
 @Composable
 internal fun ColorSample() {
-    val scrollState = rememberScrollState()
-    FlowRow(
+    val tokens = tokensColors
```

Comment:
nit: unnecessary variable, this can be merged
```kotlin
items(tokensColors)
```

Reply:
not here we can't as we need to be in a @Composable context (to get colors) :(

Reply:
ACK